lib/pages/class_feedback/view/class_feedback_view.dart

```diff
@@ -0,0 +1,689 @@
+import 'package:acs_upb_mobile/pages/class_feedback/model/class_feedback_answer.dart';
+import 'package:acs_upb_mobile/pages/class_feedback/service/feedback_provider.dart';
+import 'package:acs_upb_mobile/pages/classes/model/class.dart';
+import 'package:acs_upb_mobile/pages/people/model/person.dart';
+import 'package:acs_upb_mobile/pages/people/service/person_provider.dart';
+import 'package:acs_upb_mobile/widgets/autocomplete.dart';
+import 'package:acs_upb_mobile/widgets/icon_text.dart';
+import 'package:acs_upb_mobile/widgets/radio_emoji.dart';
+import 'package:acs_upb_mobile/widgets/scaffold.dart';
+import 'package:acs_upb_mobile/widgets/toast.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_feather_icons/flutter_feather_icons.dart';
+import 'package:provider/provider.dart';
+import 'package:acs_upb_mobile/generated/l10n.dart';
+import 'package:recase/recase.dart';
+
+class ClassFeedbackView extends StatefulWidget {
+  const ClassFeedbackView({Key key, this.classHeader}) : super(key: key);
+
+  final ClassHeader classHeader;
+
+  @override
+  _ClassFeedbackViewState createState() => _ClassFeedbackViewState();
+}
+
+class _ClassFeedbackViewState extends State<ClassFeedbackView> {
+  final formKey = GlobalKey<FormState>();
+  TextEditingController classController;
+  bool agreedToResponsibilities;
+
+  List<String> involvementPercentages = [];
+  String selectedTeacherName;
+  Person selectedAssistant;
+  List<Person> classTeachers = [];
+  Map<String, dynamic> feedbackQuestions = {};
+  List<String> responses = [];
+  List<Map<int, bool>> initialValues = [];
+
+  @override
+  void initState() {
+    super.initState();
+
+    agreedToResponsibilities = false;
+    classController = TextEditingController(text: widget.classHeader?.id ?? '');
+    involvementPercentages = [
+      '0% ... 20%',
+      '20% ... 40%',
+      '40% ... 60%',
+      '60% ... 80%',
+      '80% ... 100%'
+    ];
+    Provider.of<PersonProvider>(context, listen: false)
+        .fetchPeople()
+        .then((teachers) => setState(() => classTeachers = teachers));
+
+    awaitFeedbackQuestions();
+  }
+
+  Future<Map<String, dynamic>> awaitFeedbackQuestions() async {
+    await Provider.of<FeedbackProvider>(context, listen: false)
+        .fetchQuestions()
+        .then((questions) => setState(() => feedbackQuestions = questions));
+
+    for (int i = 0; i <= feedbackQuestions.length; i++) {
+      responses.insert(i, '-1');
+      initialValues.insert(i, {
+        0: false,
+        1: false,
+        2: false,
+        3: false,
+        4: false,
+      });
+    }
+
+    return feedbackQuestions;
+  }
+
+  Widget autocompleteAssistant(BuildContext context) {
```

Comment:
Don't just copy paste code and change it just a bit... Make a custom widget (PersonFormField? maybe put it in pages/people/view) and reuse it both here and on the AddEventPage. You can make the things that are different parameters for that widget (e.g. `controller`, `validator` etc. in this case).